pkg/audit/audit_test.go

```diff
@@ -0,0 +1,136 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"context"
+	"crypto/rand"
+	"strconv"
+	"testing"
+
+	"storj.io/storj/pkg/pb"
+)
+
+type mockDownloader struct {
+	shares map[int]Share
+}
+
+func TestDownloadShares(t *testing.T) {
+	ctx := context.Background()
+	mockShares := make(map[int]Share)
+
+	for i, tt := range []struct {
+		stripeIndex int
+		nodeAmount  int
+		shareAmount int
+		required    int
+		total       int
+		err         error
+	}{
+		{2, 30, 30, 20, 40, nil},
+	} {
+		someData := randData(32 * 1024)
+		pointer := makePointer()
+		var nodes []*pb.Node
+		for i = 0; i < tt.nodeAmount; i++ {
+			node := &pb.Node{
+				Id:      strconv.Itoa(i),
+				Address: &pb.NodeAddress{},
+			}
+			nodes = append(nodes, node)
+		}
+
+		for i = 0; i < tt.shareAmount; i++ {
+			mockShares[i] = Share{
+				Error:       tt.err,
+				PieceNumber: i,
+				Data:        someData,
+			}
+		}
+		md := mockDownloader{shares: mockShares}
```

Comment:
This test really doesn't test anything useful, it would need to use an actual downloader to be useful. You can remove this test.

Reply:
I don't think either of my tests are really useful right now-- I was also hoping for feedback on what values I should actually be testing instead or how to structure the tests in a more beneficial way.